ci/helper-functions.sh

```diff
@@ -140,3 +145,8 @@ run_linter() {
         fi
     done
 }
+
+fetch_files() {
```

Comment:
Add a comment explaining what $1 $2 $3, try to always document things even in a bash script

Reply:
will do, sorry I haven't been doing as much documentation as I'd like it's just a ton of code lol